frontend/src/layout/navigation-3000/sidepanel/panels/access_control/AccessControlObject.tsx

```diff
@@ -0,0 +1,440 @@
+                        onConfirm={() =>
+                            void updateAccessControlMembers([{ member: organization_member, level: null }])
+                        }
+                    />
+                )
+            },
+        },
+    ]
+
+    return (
+        <div className="space-y-2">
+            <AddItemsControls
+                placeholder="Search for team members to add…"
+                onAdd={async (newValues, level) => {
+                    if (guardAvailableFeature(AvailableFeature.PROJECT_BASED_PERMISSIONING)) {
+                        await updateAccessControlMembers(newValues.map((member) => ({ member, level })))
+                    }
+                }}
+                options={addableMembers.map((member) => ({
+                    key: member.id,
+                    label: `${member.user.first_name} ${member.user.email}`,
+                    labelComponent: <UserSelectItem user={member.user} />,
+                }))}
+            />
+
+            <LemonTable columns={columns} dataSource={accessControlMembers} loading={accessControlsLoading} />
+        </div>
+    )
+}
+
+function AccessControlObjectRoles(): JSX.Element | null {
+    const { accessControlRoles, accessControlsLoading, addableRoles, rolesById, availableLevels } =
+        useValues(accessControlLogic)
+    const { updateAccessControlRoles } = useAsyncActions(accessControlLogic)
+    const { guardAvailableFeature } = useValues(upgradeModalLogic)
+
+    const columns: LemonTableColumns<AccessControlTypeRole> = [
+        {
+            title: 'Role',
+            key: 'role',
+            width: 0,
+            render: (_, { role }) => (
+                <span className="whitespace-nowrap">
+                    <LemonTableLink
+                        to={urls.settings('organization-roles') + `#role=${role}`}
+                        title={rolesById[role]?.name}
+                    />
+                </span>
+            ),
+        },
+        {
+            title: 'Members',
+            key: 'members',
+            render: (_, { role }) => {
+                return (
+                    <ProfileBubbles
+                        people={
+                            rolesById[role]?.members?.map((member) => ({
+                                email: member.user.email,
+                                name: member.user.first_name,
+                                title: `${member.user.first_name} <${member.user.email}>`,
+                            })) ?? []
+                        }
+                    />
+                )
+            },
+        },
+        {
+            title: 'Level',
+            key: 'level',
+            width: 0,
+            render: (_, { access_level, role }) => {
+                return (
+                    <div className="my-1">
+                        <SimplLevelComponent
+                            size="small"
+                            level={access_level}
```

Comment:
One thing missing here potentially is a notice to indicate if the default access level is higher than the selected level (making it clear that the default level of "editor" overrides "viewer" for example